docs/developer-docs/docs/build-an-app/test/run-a-local-chain.md

```diff
@@ -10,8 +10,10 @@ This guide explains how to run a local chain for development and testing purpose
 
 ## Prerequisites
 
-- [Go](https://golang.org/dl/) 1.22.5 or later
-- [just](https://just.systems/man/en/chapter_4.html)
+Before you start, complete following prerequisites:
+
+- [Install Go](https://golang.org/doc/install) 1.22.5 or later.
+- Install [just](https://just.systems/man/en/chapter_4.html).
 
 ## 1. Clone the Warden repository
 
```

Comment:
Should we switch the branch to the latest release? ie v0.4.2 in this case? Main is usually unstable, and building a binary directly from main is not the best practice IMO.